README.md

```diff
@@ -473,7 +473,14 @@ instance.defaults.headers.common['Authorization'] = AUTH_TOKEN;
 
 ### Config order of precedence
 
-Config will be merged with an order of precedence. The order is library defaults found in [lib/defaults.js](https://github.com/axios/axios/blob/master/lib/defaults.js#L28), then `defaults` property of the instance, and finally `config` argument for the request. The latter will take precedence over the former. Here's an example.
+The config will be merged with the following order of precedence:
+1. Library defaults found in [lib/defaults.js](https://github.com/axios/axios/blob/master/lib/defaults.js#L28)
+2. The `defaults` property of the instance
+3. The `config` argument for the request.
```

Comment:
The old paragraph is fine. Unless there is something wrong, I will leave it as it is.